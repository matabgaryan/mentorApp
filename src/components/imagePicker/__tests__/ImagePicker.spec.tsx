import React from 'react';
import ImagePickerButton  from '../index';
import { cleanup } from '@testing-library/react-native';
import { shallow } from 'enzyme';

afterEach(cleanup);

describe('ImagePickerButton component', () => {
    it('should match to snapshot Form', () => {
        const snap = shallow(<ImagePickerButton />);
        expect(snap).toMatchSnapshot();
    });
});
