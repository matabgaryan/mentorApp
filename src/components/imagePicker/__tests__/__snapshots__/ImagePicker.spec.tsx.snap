// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagePickerButton component should match to snapshot Form 1`] = `ShallowWrapper {}`;
